django_ipam/templates/admin/django-ipam/change_form.html

```diff
@@ -0,0 +1,52 @@
+{% extends "admin/change_form.html" %}
+{% load i18n admin_urls %}
+{% load static %}
+
+{% block after_related_objects %}
+    <li style="list-style-type:none">
+      <div id="graph">
+      </div>
+      <div style="padding:10px">
+        <ul style="text-align:justify;">
+          {% for i in total %}
+            {% if i in used_ip %}
+              <li style="display:inline;text-align:center;">
+                <a style='color:red;'>
+                  {{i}}
```

Comment:
please use something more meaningful than `i`
for template variables, for consistency with the rest of the openwisp2 modules, please always use this style: `{{ variable_name }}` (add spaces between brackets)